brewtils/schema_parser.py

```diff
@@ -57,10 +57,13 @@ class SchemaParser(object):
     def parse_system(cls, system, from_string=False, **kwargs):
         """Convert raw JSON string or dictionary to a system model object
 
-        :param system: The raw input
-        :param from_string: True if input is a JSON string, False if a dictionary
-        :param kwargs: Additional parameters to be passed to the Schema (e.g. many=True)
-        :return: A System object
+        Args:
```

Comment:
Converted these with some regex fun and then did my best to spot check them.  I won't be too surprised if I missed something.

Reply:
I am so glad you did this... I've been putting this off forever 😄